src/lib.ts

```diff
@@ -91,6 +92,18 @@ export const baseMeat =
     ? 275
     : 250;
 
+export function averageEmbezzlerNet(): number {
+  return ((baseMeat + 750) * meatDropModifier()) / 100;
+}
+
+export function averageTouristNet(): number {
+  return (baseMeat * meatDropModifier()) / 100;
+}
+
+export function averageEmbezzlerProfit(): number {
+  return averageEmbezzlerNet() - averageTouristNet();
```

Comment:
My concern with this is that it's the equivalent of just `750 * meatDropModifier() / 100`--drops gear is an important part of your average tourist net that often gets reduced in your average embezzler net.

Reply:
Should we call it `expected` instead of `average`?

Reply:
this is just moving the old logic from local variables to a function. if we want to to be more robust we can (and probably should) take into account drop items and item drops etc later